### ISSUE TITLE
ListCollection example crashes

### ISSUE THREAD

**Issue by pepejeria** [kharrison/CodeExamples#15]
**Project**
ListCollection

**Environment**
Please include at least the following details:
- Xcode version: Version 12.0 beta 2 (12A6163b)
- iOS version: iOS 14
- Device or simulator used to reproduce the bug: simulator

**Describe the bug**
Check out CodeExamples. Open the ListCollection example and run it. This will result in the following:

```
DiffableDataSource[68322:3360678] -[UICollectionViewListCell _updateCurrentSystemTypeAccessoryViews]: unrecognized selector sent to instance 0x7fa823f26f40
-[UICollectionViewListCell _updateCurrentSystemTypeAccessoryViews]: unrecognized selector sent to instance 0x7fa823f26f40
```

**Comment by kharrison** [comment]
Odd, I'm not seeing any problem.

**Comment by pepejeria** [comment]
Yeah, that is weird. I am also using Catalina 10.15.5 (19F101). 

<img width="876" alt="Screen Shot 2020-07-09 at 14 40 23" src="https://user-images.githubusercontent.com/1394550/87041199-2da11800-c1f2-11ea-8b0f-c9b1176dfad8.png">

**Comment by kharrison** [comment]
Me to, didn't upgrade to Big Sur. I wonder what is different?

What is the error partially hidden at the top of the console? Is it complaining about finding several versions of the framework? I wonder if it is somehow linking against the wrong SDK?

**Comment by pepejeria** [comment]
```
2020-07-09 14:47:38.146576+0200 List[97404:3977423] libMobileGestalt MobileGestaltCache.c:38: No persisted cache on this platform.
objc[97404]: Class VCWeakObjectHolder is implemented in both /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/AVConference.framework/Frameworks/ViceroyTrace.framework/ViceroyTrace (0x130730e40) and /Applications/Xcode-beta.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/System/Library/PrivateFrameworks/AVConference.framework/AVConference (0x12ff0a3f0). One of the two will be used. Which one is undefined.
2020-07-09 14:47:39.453690+0200 List[97404:3968754] -[UICollectionViewListCell _updateCurrentSystemTypeAccessoryViews]: unrecognized selector sent to instance 0x7fb29d741080
-[UICollectionViewListCell _updateCurrentSystemTypeAccessoryViews]: unrecognized selector sent to instance 0x7fb29d741080
(null)
```

**Comment by kharrison** [comment]
I don't see that in my build log and who knows if it is related? There's a two year old thread in the forums which mentions it:

[Class VCWeakObjectHolder is implemented in both](https://developer.apple.com/forums/thread/92310)

Not sure what to suggest other than trying the magic incantation to the Xcode gods:

- reset simulators
- clean build folder
- delete derived data

**Comment by pepejeria** [comment]
Yeah, that didn't work either. I will try again with the next Xcode beta.

**Comment by kharrison** [comment]
Frustrating. I'll leave this open, maybe someone else is also having the same issue

**Comment by pepejeria** [comment]
Seems to be fixed in Xcode Version 12.0 beta 3 (12A8169g)
